react-components/src/components/Architecture/CommandButton.tsx

```diff
@@ -9,6 +9,14 @@ import { useTranslation } from '../i18n/I18n';
 import { type BaseCommand } from '../../architecture/base/commands/BaseCommand';
 import { getButtonType, getDefaultCommand, getIcon, getTooltipPlacement } from './utilities';
 
+export const createCommandButton = (
+  commandConstructor: () => BaseCommand,
```

Comment:
Please remove the commandConstructor. The architecture doesn't need this feature.